src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueOrderSchemaImportActivity/providers.json

```diff
@@ -1399,7 +1330,7 @@
       },
       "ShouldBookReturnLabel": {
         "__type": "ValueOfBoolean",
-        "value": false
+        "value": "#TranssmartShouldBookReturnLabelNL#"
```

Comment:
Is there any purpose to parametrize this value? Usually, we use this pattern for sensible data, such as API key, user name, password, etc. Booking return labels or not is not quite sensible. 

Same question for the Mapping Config, the Carrier Code and the Service Level.

Reply:
I will review with @TONYFECTEAU and you.

Reply:
Ok, it will wait Monday then.

Reply:
The changes to bring this parameter to SensitiveData was reverted.